smart_contracts/contracts/profiling/host-function-metrics/src/lib.rs

```diff
@@ -148,14 +149,13 @@ pub extern "C" fn call() {
     let (contract_hash, _contract_version) =
         store_function(entry_point_name, Some(named_keys.clone()));
     // Store large function with 10 named keys, then execute it.
-    runtime::call_contract::<NamedKeys>(contract_hash, entry_point_name, runtime_args);
+    runtime::call_contract::<NamedKeys>(contract_hash, entry_point_name, runtime_args.clone());
 
     // Small function
     let small_function_name =
         "s".repeat(rng.gen_range(MIN_FUNCTION_NAME_LENGTH..=MAX_FUNCTION_NAME_LENGTH));
 
     let entry_point_name = &small_function_name;
-    let runtime_args = runtime_args! {};
```

Comment:
Was this change required?

Reply:
Yes, otherwise I get the error `0: Exec(MissingArgument { name: "bytes" })` from a subsequent contract call.